config.js

```diff
@@ -49,14 +49,14 @@ function getConfig(env) {
     case 'ci':
         return {
             networkId: 'shared-test',
-            nodeUrl: 'http://shared-test.nearprotocol.com:3030',
+            nodeUrl: 'http://shared-test.near.org:3030',
```

Comment:
`https://shared-test.near.org` not working

Reply:
Use `https://rpc.ci-testnet.near.org:3030`

Reply:
![image](https://user-images.githubusercontent.com/221614/81350926-11b5b480-9091-11ea-8665-0d2df457ffcc.png)

Reply:
https://rpc.ci-testnet.near.org